atom/common/node_bindings.cc

```diff
@@ -214,6 +214,7 @@ void NodeBindings::Initialize() {
 
   // Init node.
   // (we assume node::Init would not modify the parameters under embedded mode).
+  // NOTE: If you change this line, please ping @codebytere or @MarshallOfSound
```

Comment:
That's not a helpful comment. Please spell out what would be needed here; e.g. what either of you two would need to do if this line were changed